http/sslsocket.lua

```diff
@@ -0,0 +1,470 @@
+local TIMEOUT_INFINITY      = 500 * 365 * 86400
```

Comment:
```suggestion
local TIMEOUT_INFINITY = 500 * 365 * 86400
```

Reply:
It was copy-paste from other `sslsocket` from Cartridge repo and I forgot to correct these typos. Now fixed.